src/libcommonserver/io/iohelper_win.cpp

```diff
@@ -329,126 +331,296 @@ bool IoHelper::checkIfFileIsDehydrated(const SyncPath &itemPath, bool &isDehydra
     return IoHelper::getXAttrValue(itemPath.native(), FILE_ATTRIBUTE_OFFLINE, isDehydrated, ioError);
 }
 
+static bool setRightsWindowsApi(const SyncPath &path, DWORD permission, ACCESS_MODE accessMode, IoError &ioError,
```

Comment:
Use camelcase names for variables (pACL_old, pACL_new, path_wstr, pathw_len, pathw_c...)

Reply:
correct naming in commit https://github.com/Infomaniak/desktop-kDrive/pull/68/commits/f1535f95ef6066fd2a386f7357bce7593c517588.